index.html

```diff
@@ -985,6 +985,7 @@
      installation.
      </LI>
 
+<!--
```

Comment:
So this gets rid of:

1. The suggestion of just installing metamath.exe and set.mm for windows. Looks fine to me. It doesn't seem like an especially important technique to highlight.
2. Remove instructions for how to get gcc for MacOS. This is a good change because the instructions given are completely and utterly out of date.
3. Remove instructions for rlwrap. Although I do consider rlwrap to be an essential part of an installation (and I'd maybe reword the parts of the README.TXT which call it optional), I guess I'm OK with pointing to the README.TXT for the details.